.github/workflows/coverity.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        pg: ["11.11", "12.6"]
+        pg: ["12.6", "13.2"]
```

Comment:
We don't care about coverity on PG11 anymore?

Reply:
coverity has a rate limit so i wanted to limit the number of runs and PG11 support will be removed eventually